data/pages.yaml

```diff
@@ -6,21 +6,21 @@ ansible_pkg:
 index:
   title: Ansible community documentation
   intro: Ansible offers powerful and flexible automation that is simple, decentralized, and easy.
-  novice_heading: Get started with Ansible
+  novice_heading: Novice
```

Comment:
I prefer "Get started" to novice tbh. I'm not exactly sure how well novice translates to other languages and it might have some unintended implications. Also it feels a bit like saying "newbie" which can some times be used as a pejorative. Also "Get started" has a lot more immediacy.

Another thing is that there is no corresponding "Novice resources" page so I think a little inconsistency here is OK.